packages/generator-single-spa/src/util-module/templates/src/set-public-path.js

```diff
@@ -1,3 +1,17 @@
 import { setPublicPath } from "systemjs-webpack-interop";
 
-setPublicPath('@<%= orgName %>/<%= projectName %>');
\ No newline at end of file
+setPublicPath('@<%= orgName %>/<%= projectName %>');
+// This file exists to solve a problem with how webpack loads code splits
```

Comment:
I found it a bit weird that this comment was below the code instead of above it.

Reply:
Also, prettier might be able to format this file really nicely, especially if using `/*`. It will force certain column width.

`npx prettier set-public-path.js --write`

Reply:
I think this explanation is a bit unclear. "Url origin" could mean a few things. We already have a couple explanations of this at https://single-spa.js.org/docs/faq/#code-splits and https://github.com/joeldenning/systemjs-webpack-interop#what-is-this.

I think I'd prefer just linking to those and then updating those to be good, instead of explaining it here.

```js
/* This dynamically sets the webpack public path so that code splits work properly. See related:
* https://github.com/joeldenning/systemjs-webpack-interop#what-is-this
* https://webpack.js.org/guides/public-path/#on-the-fly
* https://single-spa.js.org/docs/faq/#code-splits
*/
setPublicPath('@<%= orgName %>/<%= projectName %>'); 
```

Reply:
Linking is 1000% better. I'll change this to match.